### DISCUSSION TITLE
Schedule scripts

### DISCUSSION THREAD

**Discussion by ghost** [Ylianst/MeshCentral#4977] [General]
I know that MC allows running of scripts on endpoints but cannot be scheduled (unless I'm missing something).

A scheduling plug in is available [here](https://github.com/ryanblenis/MeshCentral-ScriptTask) but it looks to be dead. 

Are there plans to offer scheduling in the future?

**Comment by dinger1986** [comment]
I have used that recently and did a lot of what I needed/wanted.

The other option is looking at an RMM like Tactical RMM.

**Comment by SomeGuru** [comment]
@ryanblenis You are still updating and maintaining this plugin correct?

**Reply by ryanblenis** [reply]
I check to make sure it's compatible with the latest versions from time to time, but I haven't done any updates in a while. I keep an eye out for anything breaking / important, but I haven't made any real changes in a while due to life and whatnot. I'll keep them as updated as I can.

**Reply by ghost** [reply]
@ryanblenis I presume the "enabling plugins" part goes in the MC config.json file?

**Reply by ryanblenis** [reply]
Yes, that is correct

**Reply by SomeGuru** [reply]
@RichardBurt65 Did you get it working? If not let us know and we can assist you possibly... 

-SomeGuru

**Reply by ghost** [reply]
I briefly tried but didn't take it forward. I've ended up using the hosted version of mc. Much less hassle.
